packages/css/src/link/link.scss

```diff
@@ -34,8 +32,9 @@
     text-underline-offset: var(--amsterdam-link-standalone-hover-text-underline-offset);
   }
 
-  @media screen and (min-width: $amsterdam-breakpoint-typography) {
-    font-size: var(--amsterdam-link-standalone-wide-font-size);
+  .amsterdam-theme--density-high & {
```

Comment:
I mentioned this in the description: I had to follow a different approach for the icon stylesheet. The `svg` in half the selectors would result in ascending specificity if we kept the current order. We have a Stylelint rule in place to prevent specificity bugs here.